README.md

```diff
@@ -1 +1,88 @@
-# angular-concepts-readme
+# Concepts
+
+## Overview
+
+There are some core concepts that every Angular developer will need to familiarise themselves with. This will teach you the core concepts that you will be using in every Angular application you make.
+
+## Objectives
+
+- Define MVC, MVVM
+- Describe two-way data-binding
+- Demonstrate updating Models and Views without two-way binding
+- Describe Angular's two-way binding with flow charts
+- Describe Dependency Injection
+- Describe Angular's Dependency Injection
+- Describe a Single Page Application and common features
+- Demonstrate server-side rendering concepts
+- Describe Single Page Applications with Angular
+
+## MVC/MVVM
+
+You might have seen the term MVC/MVVM being thrown around. These stand for Model-View-Controller/Model-View-ViewModel respectively. These is how you architecture your software. They are both quite similar, however there are a few differences -
+
+### MVC (Model-View-Controller)
```

Comment:
Feel free to use an analogy - we'll often say that MVC is like a restaurant - the model is the chef who prepares the food, or data, the view is the table, or presentation, and the controller is the waiter, passing the data, or food, between the two.

Again, this is a great place to start - start where we know, then present a new concept. This might seem simple, but any place that you can connect two things clearly is gold. Remind us about what MVC is, and then explain MVVM in that context.